TestResultSummaryService/parsers/BenchmarkMetricRouter.js

```diff
@@ -17,13 +35,85 @@ const BenchmarkMetricRouter = {
     "LibertyStartupDT": {
         "17dev-4way-0-256-qs": "LibertyStartup"
     },
+    "LibertyStartupHugeEJB": {
+        "9dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-0-256-qs": "LibertyStartup"
+    },
+    "CleanedLibertyStartup": {
+        "9dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-0-256-qs": "LibertyStartup"
+    },
+    "CleanedLibertyStartupDT": {
+        "9dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-0-256-qs": "LibertyStartup"
+    },
+    "CleanedLibertyStartupDT7": {
+        "9dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-0-256-qs": "LibertyStartup"
+    },
+    "CleanedLibertyStartupHugeEJB": {
+        "9dev-4way-0-256-qs": "LibertyStartup",
+        "17dev-4way-0-256-qs": "LibertyStartup"
+    },
     "ILOG_WODM": {
         "881-4way-Seg5FastpathRVEJB": "ILOG_WODM",
-        "881-4way-Seg300RulesFastpathRVEJB": "ILOG_WODM"
+        "881-4way-Seg300RulesFastpathRVEJB": "ILOG_WODM",
+        "4way-Seg5FastpathRVEJB": "ILOG_WODM",
+        "4way-Seg300RulesFastpathRVEJB": "ILOG_WODM"
     },
-    "SPECjbb2015": {
+    "SPECjbb2015GMR": {
         "multi_2grp_gencon": "SPECjbb2015"
     },
+    "LibertyDayTrader3SSL": {
+        "4TLS_ECDHE_ECDSA_AES_128_CBC_SHA256_NSR": "LibertySSL",
+        "4TLS_ECDHE_ECDSA_AES_128_CBC_SHA256_SR": "LibertySSL",
+        "4TLS_ECDHE_ECDSA_AES_128_GCM_SHA256_NSR": "LibertySSL",
+        "4TLS_ECDHE_ECDSA_AES_128_GCM_SHA256_SR": "LibertySSL",
+        "4TLS_ECDHE_RSA_AES_128_CBC_SHA256_SR": "LibertySSL",
+        "4TLS_ECDHE_RSA_AES_128_CBC_SHA256_NSR": "LibertySSL",
+        "4TLS_RSA_AES_128_CBC_SHA256_NSR": "LibertySSL",
+        "4TLS_RSA_AES_128_CBC_SHA256_SR": "LibertySSL"
+    },
+    "Crypto-BB":{
+        "default": "Crypto-BB",
+    },
+    "Crypto-BB-v3":{
+        "default": "Crypto-BB-v3",
+        "IBMJCEPlus": "Crypto-BB-v3"
+    },
```

Comment:
Parsers are missing of them. Let's leave them out for now, and we can add in another PR.

Reply:
Also, make sure things don't break if the parser is missing.